shared/common-adapters/avatar.js

```diff
@@ -109,6 +109,8 @@ class SharedAskForUserData {
   _userQueue = {}
   _userLastReq = {}
   _username = ''
+  _teamQueue: Object = {}
+  _userQueue: Object = {}
 
```

Comment:
Both of these already in there, this one just has a type on it. Delete the two above it?